app/views/layouts/angular-bootstrap/_auth_credentials_angular_bootstrap.html.haml

```diff
@@ -32,7 +33,7 @@
       %input.form-control{"type"         => "password",
                           "id"           => "textInput-markup",
                           "ng-required"  => "#{ng_reqd_password}",
-                          "placeholder"  => "#{placeholder_if_present("#{ng_model}.#{prefix}_password")}",
+                          "placeholder"  => "",
```

Comment:
Isn't the whole point of the placeholder_if_present method to do exactly what you are doing in this PR?  cc @martinpovolny